crates/matrix-sdk/src/media.rs

```diff
@@ -96,6 +113,43 @@ impl Media {
         Ok(self.client.send(request, Some(request_config)).await?)
     }
 
+    /// Gets a media file by copying it to a temporary location on disk.
+    ///
+    /// If the content is encrypted and encryption is enabled, the content will
+    /// be decrypted.
```

Comment:
I think it might be clearer to say "the file won't be encrypted even if it is encrypted on the server".